libs/shared/styles/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './lib/shared-styles';
```

Comment:
Not sure on our policy about copyright headers.
I know a bunch of legacy files don't have them and it's probably a lost battle, but ideally add copyrights to new files.

Reply:
Going to update as per @bbangert comment on another PR: https://github.com/mozilla/fxa/pull/15627#discussion_r1282011108

Reply:
If its a Boring Peter Grep(tm) you're wanting&hellip; _today's your lucky day!_

```sh
git grep -L "This Source Code Form is subject to the terms of the Mozilla Public" apps/payments/**/*.{ts,tsx} | cat

apps/payments/next/app/api/hello/route.ts
apps/payments/next/app/layout.tsx
apps/payments/next/app/page.tsx
apps/payments/next/components/ExampleHeader/index.test.tsx
apps/payments/next/components/ExampleHeader/index.tsx
apps/payments/next/components/HelloWorld/index.stories.tsx
apps/payments/next/components/HelloWorld/index.tsx
apps/payments/next/index.d.ts
apps/payments/next/jest.config.ts
apps/payments/next/next-env.d.ts
apps/payments/next/tests/app/page.test.tsx
```

(then just adjust the glob to your liking)